src/Sentry.AspNetCore/FodyWeavers.xsd

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Anyv luck with the symlinks? These 300 lines of xml make me nervous

Reply:
Will look into this next

Reply:
I seem to have created a symlink (using `mklink` on Windows tho), but not sure how git is supposed to handle that. Reading [here](https://stackoverflow.com/questions/954560/how-does-git-handle-symbolic-links), it seems that git still handles them as separate files (which seems to be confirmed by `git diff`):
![image](https://user-images.githubusercontent.com/1935960/96281747-72eb9780-0fe2-11eb-8691-3e23bf3c3160.png)

Reply:
Nevermind, apparently I was creating a hardlink and not a symlink (no idea what the difference is, but it seems to work now).